### PR TITLE
no matter the ekco version, only show an info message

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: pkg/lint/rego/output.rego [hunk 1]
```diff
@@ -246,6 +246,7 @@ lint[output] {
 	some name 
 	ignored := known_versions[name]
 	not valid_add_on_version(name)
+	name != "ekco"
```
Comment (emosbaugh) [inline pkg/lint/rego/output.rego:249]: is this only true for kurl versions >= v2023.04.11-0

**Review by emosbaugh — CHANGES_REQUESTED** [review]

File: pkg/lint/rego/output.rego (hunk 1 above)
Comment (laverya) [reply pkg/lint/rego/output.rego:249]: not at the moment unfortunately

Comment (laverya) [reply pkg/lint/rego/output.rego:249]: I can try to add that logic but it'll take some work

Comment (emosbaugh) [reply pkg/lint/rego/output.rego:249]: i will approve as this prevents false positives

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**